src/script/components/InputBar/TypingIndicator/TypingIndicator.styles.ts

```diff
@@ -22,18 +22,19 @@ import {css, CSSObject, keyframes} from '@emotion/react';
 export const wrapperStyles: CSSObject = {
   display: 'flex',
   alignItems: 'center',
-  marginBottom: '12px',
-  marginLeft: '18px',
+  marginBottom: 5,
+  marginLeft: 18,
   color: 'var(--text-input-placeholder)',
-  fontSize: '14px',
+  fontSize: 11,
```

Comment:
```suggestion
  fontSize: @font-size-xsmall,
```
one variable less for my next pr ;)

Reply:
this is typescript, it's gonna break this way